Detections/SecurityEvent/RDP_MultipleConnectionsFromSingleSystem.yaml

```diff
@@ -25,16 +25,43 @@ query: |
   let endtime = 1d;
   let starttime = 8d;
   let threshold = 2.0;
-  SecurityEvent
+  (union isfuzzy=true
+  (SecurityEvent
   | where TimeGenerated >= ago(endtime)
   | where EventID == 4624 and LogonType == 10
   | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), ComputerCountToday = dcount(Computer), ComputerSet = makeset(Computer), ProcessSet = makeset(ProcessName)
-  by Account, IpAddress, AccountType, Activity, LogonTypeName
+  by Account, IpAddress, AccountType, Activity, LogonTypeName),
+  (WindowsEvent
+  | where TimeGenerated >= ago(endtime)
+  | where EventID == 4624 
```

Comment:
I would add where EventData has "10". Well, you actually did it in the next one :-)